static/styles/rendered_markdown.css

```diff
@@ -52,12 +52,26 @@
     h4,
     h5,
     h6 {
-        font-size: 14px;
+        font-size: 1em;
         font-weight: 600;
         line-height: 1.4;
-        margin-top: 0;
-        margin-bottom: 5px;
-        text-decoration: underline;
+        margin-top: 14px;
```

Comment:
Can we make this margin 1 line?  That'll be useful when we migrate to support multiple font sizes.

Reply:
For sure, sounds great.